### DISCUSSION TITLE
v0.18.2 (Fixes notifications)

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1293] [Releases]
## What's Changed
* [Bug] Don't try to send empty threshold notifications by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1291
* Release v0.18.2 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1292

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.18.1...v0.18.2

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.18.2'>v0.18.2 (Fixes notifications)</a>.</em>

**Comment by seh2000** [comment]
Updated, and my error related to email gone :)
